valet-key/ValetKey.Client/Program.cs

```diff
@@ -24,19 +23,17 @@ public static void Main(string[] args)
             try
             {
                 var blobSas = GetBlobSas(new Uri(tokenServiceEndpoint)).Result;
```

Comment:
Can we use the new BlobSasBuilder instead of GetBlobSas custom method?

Reply:
The new BlobSasBuilder logic is in the WebApp. Here, in client side app,  it just get BlobSasToken  via requesting the the api exposed by WebApp.